contracts/utils/BridgeMessage.sol

```diff
@@ -216,11 +219,15 @@ library BridgeMessage {
         pure
         returns (uint8, uint256)
     {
+    // 1. could price smaller uint?
+    // 2. should it be abi.decodePacked()
+    // 3. do we want uint256 or whatever type to be big endian?
```

Comment:
1. possibly uint64?
2. no
3. big endian from here on out is preferable.

Reply:
passing the order of big endian to @patrickkuo lol